api.py

```diff
@@ -50,6 +50,9 @@ def send_api_request(self, query_info: dict) -> dict:
             dict: API response from request.
         """
         headers = self._get_headers()
-        req_data = json.dumps(query_info)
-        res = requests.post(urljoin(API_URL, 'graphql'), headers=headers, data=req_data)
-        return json.loads(res.text)
+        res = requests.post(urljoin(API_URL, 'graphql'), headers=headers, json=query_info)
+        resp_value = res.json()
+        if resp_value.get('errors'):
+            print('---AN ERROR OCCURRED IN THE API REQUEST---')
+            print(resp_value)
```

Comment:
Not a big deal but maybe we should get the error message from the response not print the whole JSON doc? Also we should use a logger instead of print!

Reply:
done! I left it as the whole thing in this so it's raw. It's just an example. SDK would be more robust when we have an SDK!